app/assets/stylesheets/sufia/admin.scss

```diff
@@ -33,6 +33,10 @@ body {
   background-color: $admin-sidebar-background-color;
 }
 
+fieldset {
```

Comment:
I suspect that all the containers on that page have `float: left` except for that fieldset.  Would be up for experimenting to see if that fixes it for you? Otherwise add a comment about this being needed for the admin/admin_set_controller on Chrome.